scripts/lldpshow

```diff
@@ -115,11 +168,31 @@ class Lldpshow(object):
             print ('Error:',self.err)
 
 def main():
+    parser  = argparse.ArgumentParser(description='Display the LLDP neighbors',
+                                      version='1.0.0',
+                                      formatter_class=argparse.RawTextHelpFormatter,
+                                      epilog="""
+                                      Examples:
+                                      lldpshow
+                                      lldpshow -d
+                                      lldpshow -d -p Ethernet0
+                                      """)
+
+    parser.add_argument('-d', '--detail', action='store_true', help='LLDP neighbors detail information', default=False)
+    parser.add_argument('-p', '--port', type=str, help='LLDP neighbors detail information for given port', default=None)
+    args = parser.parse_args()
+
+    lldp_detail_info = args.detail
+    lldp_port = args.port
+
+    if lldp_port and not lldp_detail_info:
+        parser.error('-d is required when -p is set.')
```

Comment:
why do we need this? Can we set -d automatically when -p?

Reply:
@pavel-shirshov yes that is fine too. Updated accordingly.

Reply:
@pavel-shirshov can you please review/approve the PR.  Above comment has been addressed.